config/middleware.py

```diff
@@ -72,9 +72,9 @@ def process_template_response(self, request, response):
         if "feedback_survey_type" in response.context_data:
             params["type"] = response.context_data["feedback_survey_type"]
 
-        if "feedback_survey_judgment_uri" in response.context_data:
+        if "feedback_survey_document_uri" in response.context_data:
             params["judgment_uri"] = response.context_data[
```

Comment:
Does this line also want tweaking?

Reply:
I think not yet, I should have aded the comment 

```
# TODO: update the survey to allow for generalisation to `document`
            # https://trello.com/c/l0iBFM1e/1151-update-survey-to-account-for-judgment-the-fact-that-we-have-press-summaries-as-well-as-judgments-now
```

like I did for https://github.com/nationalarchives/ds-caselaw-public-ui/pull/815/files#diff-5c9236772e72d720d3783256e03d3441839b3958cf5f891688c0ca8ae964f4d6R115

Reply:
have added the comment to here too